lib/jsmidgen.js

```diff
@@ -408,6 +408,15 @@ var Midi = {};
 		return bytes;
 	};
 
+	File.prototype.setTicks = function(num) {
+		// Do nothing in case of invalid input
+		if (!num || typeof num !== 'number') {
+			return;
+		}
+
+		this.HDR_SPEED = String.fromCharCode((num/256),  num%256);
+	}
```

Comment:
We can also use buffers and do the same as follows:
```
var speed = new Buffer(2);
speed.writeInt16BE(num);
this.HDR_SPEED = String.fromCharCode(speed[0], speed[1]);
```
I did not use this `Buffer` based approach directly as we are exporting the module for the browser as well. But this seems like the least hacky way so far.